<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>blnuhr</class>
 <widget class="QWidget" name="blnuhr">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>246</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>blnuhr</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="blinker_layout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="l_s">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(255, 170, 127);
color: rgb(255, 170, 127);</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="h5_layout">
     <item>
      <widget class="QLabel" name="l_h5_1">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 85, 127);</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="l_h5_2">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 85, 127);</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="l_h5_3">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(128, 42, 64);</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="l_h5_4">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(128, 42, 64);</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="h1_layout">
     <item>
      <widget class="QLabel" name="l_h1_1">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 85, 127);</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="l_h1_2">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 85, 127);</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="l_h1_3">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(128, 42, 64);</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="l_h1_4">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(128, 42, 64);</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="m5_layout">
     <item>
      <widget class="QLabel" name="l_m5_1">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 170, 127);</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="l_m5_2">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 170, 127);</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="l_m5_3">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 85, 127);</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="l_m5_4">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 170, 127);</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="l_m5_5">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(130, 85, 64);</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="l_m5_6">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(128, 42, 64);</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="l_m5_7">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(130, 85, 64);</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="l_m5_8">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(130, 85, 64);</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="l_m5_9">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(128, 42, 64);</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="l_m5_10">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(130, 85, 64);</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="l_m5_11">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(130, 85, 64);</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="m1_layout">
     <item>
      <widget class="QLabel" name="l_m1_1">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 170, 127);</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="l_m1_2">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 170, 127);</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="l_m1_3">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(130, 85, 64);</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="l_m1_4">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(130, 85, 64);</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="s5_layout">
     <item>
      <widget class="QLabel" name="l_s5_1">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 170, 127);</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="l_s5_2">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 170, 127);</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="l_s5_3">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 85, 127);</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="l_s5_4">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 170, 127);</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="l_s5_5">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 170, 127);</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="l_s5_6">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(128, 42, 64);</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="l_s5_7">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(130, 85, 64);</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="l_s5_8">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(130, 85, 64);</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="l_s5_9">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(128, 42, 64);</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="l_s5_10">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(130, 85, 64);</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="l_s5_11">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(130, 85, 64);</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="s1_layout">
     <item>
      <widget class="QLabel" name="l_s1_1">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 170, 127);</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="l_s1_2">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 170, 127);</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="l_s1_3">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(130, 85, 64);</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="l_s1_4">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(130, 85, 64);</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
